docs/content/Gameplay/mechanics/tools/Gadgets/Terminal.md

```diff
@@ -1,3 +1,12 @@
 ---
 title: Terminal
 ---
+# The Terminal
+
+The terminal is used to assemble multiblocks automatically.
+
+Shift right-clicking a multiblock controller with the terminal will place all blocks it can from your inventory, prioritizing using hatches/multiparts until it can place the minimum casing.
+
+It is recommended to auto-build multiblocks with just their casing and other non multipart blocks in your inventory, then place your multiparts, as multiparts like hatches will likely not be placed where you want them to be.
```

Comment:
"multipart" seems like a weird term to use (possibly one used internally) for this.

I wonder if the terminal should have a mode where it ignores those blocks?

Reply:
Yea the terminal should prob have a mode that or by default doesn't place auto predicates